src/main/java/com/kodilla/ecommercee/mapper/UserMapper.java

```diff
@@ -24,7 +24,7 @@ public UserDto mapToUserDto(final User user) {
         return new UserDto(
                 user.getId(),
                 user.getName(),
-                user.getPassword(),
+                user.getPassword().replaceAll("[^a-zA-Z0-9]", "").replaceAll("[a-zA-Z0-9]", ""),
```

Comment:
Działa. Można by rozważyć sugestię Michała i zrobić 2 Dto - jeden jako input (dodajemy użytkownika) i drugi output do pobierania danych użytkowników.

Reply:
Poprawie, to był mój autorski pomysł :)